.github/workflows/release-state-manager.yml

```
@@ -44,30 +48,27 @@ jobs:
         working-directory: state-manager
         run: |
           uv sync --group dev
-
-      - name: Run unit tests with pytest and coverage
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)